rand_distr/src/frechet.rs

```diff
@@ -6,20 +6,36 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! The Fréchet distribution.
+//! The Fréchet distribution `Fréchet(α, μ, σ)`.
 
 use crate::{Distribution, OpenClosed01};
 use core::fmt;
 use num_traits::Float;
 use rand::Rng;
 
-/// Samples floating-point numbers according to the Fréchet distribution
+/// The [Fréchet distribution](https://en.wikipedia.org/wiki/Fr%C3%A9chet_distribution) `Fréchet(α, μ, σ)`.
 ///
-/// This distribution has density function:
-/// `f(x) = [(x - μ) / σ]^(-1 - α) exp[-(x - μ) / σ]^(-α) α / σ`,
-/// where `μ` is the location parameter, `σ` the scale parameter, and `α` the shape parameter.
+/// The Fréchet distribution is a continuous probability distribution
+/// with shape parameter `α` (`alpha`), location parameter `μ` (`mu`),
+/// and scale parameter `σ` (`sigma`). It describes the distribution
+/// of the maximum (or minimum) of a number of random variables.
+/// It is also known as the Type II extreme value distribution.
```

Comment:
The constructor uses argument order `location, scale, shape`; I suggest we do the same here.

Reply:
Oh, good eye!